www/templates/user-vhost-warning

```diff
@@ -1,23 +1,25 @@
-Dear {group}:
+Hi {group},
 
-The OCF staff noticed that your group's website [1] does not have either
-the "Hosted by OCF" banner [2] or University disclaimer [3] required by OCF
-and university virtual hosting rules. All student group websites are
-required to display the "Hosted by OCF" banner as well as the student group
-disclaimer. Unfortunately, if you do not add both of them, we may be forced
-to temporarily disable your website to comply with the OCF's and
-university's requirements. Please add both sometime within the next 2 weeks
-to prevent disabling of your website.
+The staff at the OCF noticed that your group's website [1] may be missing a
+"Hosted by OCF" banner [2] or the University-mandated "student group
+disclaimer" [3]. All student group websites must display both elements to
+comply with OCF virtual hosting policy and University trademark
+regulation. If your website isn't fully compliant, we will have to
+temporarily disable it to comply with these requirements. We'll check back
+in about two weeks, so please try to make any needed changes by then.
 
 Your username is "{username}".
 
-If you've forgotten how to login, instructions can be found here [4]. If
+If you've forgotten how to log in, instructions can be found here [4]. If
 you've forgotten your password, you can reset it here [5].
 
 Feel free to reply all to this email if you have any questions or if we can
 help with making the change! Please include help@ocf.berkeley.edu if you
 would like a response.
```

Comment:
i still think we don't need to tell them to "please include help@", we should be either sending with RT, or setting Reply-To so the reply goes to RT (preferably the former)

Reply:
Very true. I'm not sure if that can be incorporated into the template in any way, so presumably we will just rely on staff to practice good habits, or make a remark somewhere in the docs.